Added missing .UseApplicationInsights() in VotingWeb

fbeltrao opened the pull request:
Following the step by step guide will not collect information from VotingWeb because the demo code does not include a call to UseApplicationInsights(). VotingData does have it.

I think it is important to emphasis the need of calling it.

I will additionally make a PR to fix the problem in https://github.com/Azure-Samples/service-fabric-dotnet-quickstart

azurecla commented:
Hi __@TheBeardedLlama__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@CawaMS - @TheBeardedLlama proposed a new step in this PR - please review.

CawaMS commented:
For the purpose of this tutorial, we intended that the entire build should fail if any step fails so no time will be wasted on the subsequent steps. The 'Continue On Error' is a valid scenario that users can decide to enable themselves. Though it's not really mandatory to complete the tasks here.

CawaMS commented:
#sign-off

TheBeardedLlama commented:
Hi @CawaMS, I believe there has been a misunderstanding.

The build will *always* fail if continue on error is not enabled, because the PS script will throw an error if the resource group does not exist. The resource group should not exist based on the instructions themselves, so the build will *always* fail.

tysonn commented:
@mthalman-msft - Hey Matt, could you look at this public PR now that you've taken over this article?

mthalman-msft commented:
@tysonn: Looks good.

fbeltrao committed:
Added missing .UseApplicationInsights() in VotingWeb

Following the step by step guide does not collection information from VotingWeb because the demo code does not include a call to UseApplicationInsights(). I think it is important to emphasis the need of calling it.
I will additionally make a PR to fix the problem in https://github.com/Azure-Samples/service-fabric-dotnet-quickstart
